core/src/main/scala/stryker4s/mutants/Mutator.scala

```diff
@@ -44,12 +44,17 @@ class Mutator(mutantFinder: MutantFinder, transformer: StatementTransformer, mat
   private def logMutationResult(mutatedFiles: Iterable[MutatedFile], totalAmountOfFiles: Int): Unit = {
     val includedMutants = mutatedFiles.flatMap(_.mutants).size
     val excludedMutants = mutatedFiles.map(_.excludedMutants).sum
+    val totalMutants = includedMutants + excludedMutants
 
     info(s"Found ${mutatedFiles.size} of $totalAmountOfFiles file(s) to be mutated.")
-    info(s"${includedMutants + excludedMutants} Mutant(s) generated.")
+    info(s"$totalMutants Mutant(s) generated.")
     if (excludedMutants > 0) {
       info(s"Of which $excludedMutants Mutant(s) are excluded.")
     }
+    if (includedMutants == 0) {
+      info("It's a mutant-free world, nothing to test.")
+      info("If this is not intended, please check your configuration and try again.")
```

Comment:
Maybe we should add some hints on what could be wrong for example:
* Amount of generated files = 0 so file collection might be wrong or the basepath
* Files are found but nothing is mutated maybe everything is excluded

etc.

Reply:
Good point 👍. I'll see if I can rewrite it a little bit tomorrow

Reply:
I've changed the logs a bit. It now logs more about scenarios that could be problematic and provides a hint as to what configuration could be the cause.